tests/performance/common/auth.py

```diff
@@ -1,78 +0,0 @@
-import requests
```

Comment:
Performance testing is used for load testing using locust.. Shall you keep this since is not related with your integration tests?